### PR TITLE
Add isMobile value

### PR DESCRIPTION
**Pull request by sbrunner**
fix: #437

### CODE REVIEW

File: contribs/gmf/apps/mobile/js/mobile.js [hunk 1]
```diff
@@ -24,6 +24,9 @@ goog.require('ol.layer.Tile');
 goog.require('ol.source.OSM');
 
 
+gmfModule.value('isMobile', true);
```
Comment (gberaudo) [inline contribs/gmf/apps/mobile/js/mobile.js]: Is the value constant? In this case you may call `gmfModule.constant()` instead.

Comment (sbrunner) [inline contribs/gmf/apps/mobile/js/mobile.js]: Thanks, fixed :-)

**Comment by gberaudo** [comment]
LGTM

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**